fluent-plugin-validate-tag-filter.gemspec

```diff
@@ -21,5 +21,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.10"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "test-unit"
-  spec.add_runtime_dependency "fluentd"
+  spec.add_runtime_dependency "fluentd", [">= 0.14.0", "< 2"]
```

Comment:
Why do you limit upper version `2` ?
[Migration guide](https://docs.fluentd.org/v0.14/articles/plugin-update-from-v12) recommends to use `>= 0.14.0`, but I don't see about upper version. Is this upper version necessary?

Reply:
This is because planned Fluentd versioning:

v0.14.x will be stable and just bump up version as v1.
v2 will introduce non-backward compatible change.

This patch considers future breaking changes.

Reply:
👍